askup/tests/test_functional/test_general.py

```diff
@@ -808,3 +812,203 @@ def test_downvote_question_fail_permission(self):
         self.downvote_question(question_id)
         result_votes = Vote.objects.filter(question_id=question_id).aggregate(models.Sum('value'))
         self.assertEqual(result_votes['value__sum'], original_votes['value__sum'])
+
+
+class TestUserSignUp(LoginAdminByDefaultMixIn, TestCase):
+    """Tests the user sign up process."""
+
+    fixtures = ['groups', 'mockup_data']
+
+    def setUp(self):
+        """
+        Set up a TestCase.
+        """
+        settings.DEBUG = False
+        self.default_login()
```

Comment:
login could be omitted in following tests

Reply:
done